app/views/users/attendance_report.html.erb

```diff
@@ -0,0 +1,45 @@
+<h1>Attendance Report</h2>
+
+<table>
+  <thead>
+    <tr>
+      <th>User ID</th>
+      <th>First name</th>
+      <th>Last name</th>
+      <th>Attendance Credits</th>
+      <th>Requirement Met?</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <% if User.any? %>
+  <tbody>
+    <% User.all.each do |user| %>
+      <tr>
+        <td><%= user.id %></td>
+        <td><%= user.first_name %></td>
+        <td><%= user.last_name %></td>
+        <% # checks RSVPS that include the email of the user being checked to add a credit for each one %>
+        <td><%= Rsvp.where(user_id: user.email).count %></td>
```

Comment:
Shouldn't the attendance also be checked here?
Rsvp.where(user_id: user.email, attendance: true)